tests/topotests/bgp_rpki_topo1/r1/rtrd.py

```diff
@@ -0,0 +1,271 @@
+#!/usr/bin/python3
+#
+# by Tomas Hlavacek (tmshlvck@gmail.com)
+
+import socket
+import threading
+import socketserver
+import struct
+import ipaddress
+
+HOST, PORT = "0.0.0.0", 15432
+
+
+def dbg(m):
+    print(m)
+
+
+class RTRDatabase(object):
+    def __init__(self):
+        self.last_serial = 0
+
+    def get_serial(self):
+        return self.last_serial
+
+    def set_serial(self, serial):
+        self.last_serial = serial
+
+    def get_announcements4(self, serial=0):
+        if serial > self.last_serial:
+            return [(29134, "217.31.48.0/20", 20), (29134, "62.109.128.0/19", 19)]
```

Comment:
What are these ASNs and prefixes?

Reply:
changed to private TEST-2 and TEST-3 prefixes and to a private ASN